src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Controls/Given_ScrollViewer.cs

```diff
@@ -605,5 +608,97 @@ public async Task When_DoublyNested_BringIntoView()
 		}
 
 #endif
+
+		[TestMethod]
+		[RunsOnUIThread]
+#if !__SKIA__
+		[Ignore("Pointer injection supported only on skia for now.")]
+#endif
+		public async Task When_TouchScroll_Then_NestedElementReceivePointerEvents()
+		{
+			if (Private.Infrastructure.TestServices.WindowHelper.IsXamlIsland)
+			{
+				Assert.Inconclusive("Pointer injection is not supported yet on XamlIsland");
+				return;
+			}
+
+			var nested = new Border
+			{
+				Height = 4192,
+				Width = 256,
+				Background = new SolidColorBrush(Colors.DeepPink)
+			};
+			var sut = new ScrollViewer
+			{
+				Height = 512,
+				Width = 256,
+				Content = nested
+			};
+
+			var events = new List<string>();
+			nested.PointerEntered += (snd, e) => events.Add("enter");
+			nested.PointerExited += (snd, e) => events.Add("exited");
+			nested.PointerPressed += (snd, e) => events.Add("pressed");
+			nested.PointerReleased += (snd, e) => events.Add("release");
+			nested.PointerCanceled += (snd, e) => events.Add("cancel");
+
+			WindowHelper.WindowContent = new Grid { Children = { sut } };
+			await WindowHelper.WaitForLoaded(nested);
+			await WindowHelper.WaitForIdle();
+
+			var input = InputInjector.TryCreate() ?? throw new InvalidOperationException("Pointer injection not available on this platform.");
+			using var finger = input.GetFinger();
+
+			var sutLocation = sut.GetAbsoluteBounds().Location;
+			finger.Drag(sutLocation.Offset(5, 480), sutLocation.Offset(5, 5));
+
+			events.Should().BeEquivalentTo("enter", "pressed", "release", "exited"); // TODO: Exited is not injected by the InputInjector (but it is with native)
```

Comment:
Do you have an issue or task that you can link for this TODO ;) ?

Reply:
Well actually the comment is no longer valid, I fixed it :)
```suggestion
			events.Should().BeEquivalentTo("enter", "pressed", "release", "exited");
```